pynuki/opener.py

```diff
@@ -10,6 +10,14 @@ class NukiOpener(NukiDevice):
     def is_rto_activated(self):
         return self.state == const.STATE_OPENER_RTO_ACTIVE
 
+    @property
+    def ringactionTimestamp(self):
```

Comment:
Please use snake case


```suggestion
    def ring_action_timestamp(self):
```

Reply:
done :)